src/common/components/new-tab-link-with-tooltip.tsx

```diff
@@ -6,12 +6,15 @@ import * as React from 'react';
 import { NamedFC } from '../react/named-fc';
 import styles from './new-tab-link-with-tooltip.scss';
 
-export type NewTabLinkWithTooltipProps = ILinkProps & { tooltipContent: string | undefined };
+export type NewTabLinkWithTooltipProps = ILinkProps & {
+    tooltipContent: string | undefined;
+    className?: string | undefined;
```

Comment:
is className optional value?

Reply:
Yes, we are keeping className as optional because we have to use specific className (insightsLinkIcon) for icon that we are passing but for rest by default className (insightsLink) is being picked which is as expected.